apps/meteor/client/hooks/omnichannel/useIsRoomOverMacLimit.tsx

```diff
@@ -4,6 +4,7 @@ import { isOmnichannelRoom, type IOmnichannelGenericRoom, isVoipRoom } from '@ro
 import { useIsOverMacLimit } from './useIsOverMacLimit';
 
 const getPeriod = (date: Date) => `${date.getFullYear()}-${String(date.getMonth() + 1).padStart(2, '0')}`;
+const isILivechatInquiry = (room: any) => room.rid && room.status;
```

Comment:
few things
- dont use `any`  
- do not create a "typeguard" that doest return a type 
- I dont think this makes sense, if you are really passing IRoom, ILivechatInquiryRecord should inherit Iroom. maybe consider creating a new hook or reconsider the usage